src/metax_api/migrations/0026_auto_20200917_1615.py

```diff
@@ -40,7 +41,6 @@ def change_emails(apps, schema_editor):
         research_dataset_changed = False
 
         dataset = json.loads(json.dumps(cr.research_dataset))
-        #_logger.warning(f'DATASET TYPE IS {type(dataset)}')
 
```

Comment:
minor remark: might be good convention to transform logger message types to debug instead of commenting out. This way if someone else needs to debug the same code, they can turn on the debug logging level and consume previous logging messages instantly.

Reply:
Good point, I wrote this down to my notes :D :)